engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/impl/history/UserOperationLogRestServiceImpl.java

```diff
@@ -64,4 +65,19 @@ public CountResultDto queryUserOperationCount(UriInfo uriInfo) {
       return UserOperationLogEntryDto.map(query.listPage(firstResult, maxResults));
     }
   }
+
+  @Override
+  public void setAnnotation(String operationId, AnnotationDto annotationDto) {
+    String annotation = annotationDto.getAnnotation();
+
+    processEngine.getHistoryService()
+        .setAnnotationForOperationLogById(operationId, annotation);
+  }
```

Comment:
let's return a `Response` object with `noContent` instead of `void`, so we're able to change what we return but stay backwards-compatible